frontend/appflowy_flutter/lib/mobile/presentation/bottom_sheet/show_mobile_bottom_sheet.dart

```diff
@@ -166,8 +163,9 @@ Future<T?> showMobileBottomSheet<T>(
   );
 }
 
-class _Header extends StatelessWidget {
-  const _Header({
+class Header extends StatelessWidget {
```

Comment:
If this widget is public, rename it. The name `Header` is too common and likely to cause conflicts.

Reply:
Fixed.